DOWNLOAD_SNAPSHOT_JAR.md

```diff
@@ -0,0 +1,23 @@
+# Instructions to download the latest snapshot version of the validator
```

Comment:
This is a lot of steps to follow to get the latest SNAPSHOT, including some manual steps that are error prone (with two actions having the same name).

Previously we used Travis to build a ` gtfs-realtime-validator-lib-v1.0.0-SNAPSHOT` and ` gtfs-realtime-validator-webapp-v1.0.0-SNAPSHOT` and push them to a public Amazon S3 bucket on each commit (this predated GitHub Actions). This let us put a stable URL on the README where someone could always access the latest SNAPSHOT with a single click.

Can we do something similar here to have a single click to get the latest SNAPSHOT? Is there a GitHub shortcut/redirect to the latest run or artifact for a particular action?

Having the hash and branch name in the JAR name is nice as it makes it simple to tell it's origin, but it definitely makes the above more complicated.

Reply:
This is definitely something we could do. Should it be added as a nice to have in another issue/PR? The process described here is the same as the GTFS Schedule validator one. What are your thoughts on this @isabelle-dr ? Either works for me.

Reply:
Hi Maxime. Thank you for working on this 🙏. 
I'm in favour of using the JAR from the latest stable release, not the master branch.
We do this in the Schedule Validator by providing a direct link to past releases in the README, and we update the README from the release with a direct link to the JAR (see below).
We just need to make sure we have releases regularly so the users aren't missing out on a functional newer version of the validator.

<img width="500" alt="Screen Shot 2022-02-17 at 9 03 36 AM" src="https://user-images.githubusercontent.com/63653518/154497563-6c9f502c-379a-4cae-86be-331cb7081e3f.png">

That being said, having a stable URL to the latest SNAPSHOT with a single click would be useful. I'd say we can open an issue and prioritize this modification according to the feedback we receive from users.

Reply:
No, this is good :)